src/main/java/org/kainos/ea/model/JobRoleRequest.java

```diff
@@ -0,0 +1,66 @@
+package org.kainos.ea.model;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class JobRoleRequest {
+    private String name;
+    private String specificationDesc;
+    private int capabilityId;
+    private int bandId;
+    private String url;
+
+    @JsonCreator
+    public JobRoleRequest(@JsonProperty("name") String name, @JsonProperty("specificationDesc") String specificationDesc, @JsonProperty("capabilityId") int capabilityId, @JsonProperty("bandId") int bandId, @JsonProperty("url") String url) {
```

Comment:
Does splitting this across multiple lines make it easier to read? What do you think?

```suggestion
    public JobRoleRequest(
        @JsonProperty("name") String name, 
        @JsonProperty("specificationDesc") String specificationDesc, 
        @JsonProperty("capabilityId") int capabilityId,
        @JsonProperty("bandId") int bandId,
        @JsonProperty("url") String url) {
```